File: 1007 - diferenca/dif.java
import java.util.Scanner;
import java.io.IOException;

public class Main{

    public static int diferenca( int a, int b, int c, int d){
        return ( ( a * b) - (c * d));
    }

    public static void main( String[] args ) throws IOException{
        Scanner input = new Scanner(System.in);
        int a, b, c, d, diff;
        a = input.nextInt();
        b = input.nextInt();
        c = input.nextInt();
        d = input.nextInt();
        diff = diferenca(a, b, c, d);
        System.out.printf("DIFERENCA = %d\n", diff);
        input.close();
    }
}

File: 1007 - diferenca/dif.kt
import java.util.Scanner
import kotlin.text.*

fun diferenca(a: Int, b: Int, c: Int, d: Int) : Int{
    return ( (a*b)-(c*d));
}

fun main(args:Array<String>){
    var input = Scanner(System.`in`)
    var a: Int; var b: Int; var c: Int;var d: Int; var diff: Int
    a = input.nextInt()
    b = input.nextInt()
    c = input.nextInt()
    d = input.nextInt()
    diff = diferenca(a, b, c, d)
    println("DIFERENCA = $diff")
}